sites/docs/src/content/docs/tutorials/tests_and_test_data/components/02_commands_integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "2. nf-test Commands & Integration"
+subtitle: Essential commands and nf-core integration
+weight: 20
+---
+
```

Comment:
I would start off by saying this is a reference page, it doesn't make sense to have these commands before the set up isntructions. Or even put this page right to the end!